README.md

```diff
@@ -18,6 +18,6 @@ Jon:
 
 
 Kirsten:
-1. Your response here.
-2. And here...
-3. Also here!
+1. I was burned in a terrible pickle accident.
```

Comment:
Aren't pickles cold?

Reply:
This seems too weird not to have happened

Reply:
im suspicious !

Reply:
How do you get burned by a pickle?